west.yml

```diff
@@ -189,7 +189,7 @@ manifest:
       groups:
         - hal
     - name: hal_nordic
-      revision: pull/265/head
+      revision: fae15426c6b5a1f67362d508cf51b691ae5ab4b4
```

Comment:
This should be done in the first commit, not here.

Reply:
given all three initial commits break if not applied together, the order is not really relevant here, I can squash them all together? its just not how I see it normally done :)

Reply:
wait, this looks wrong, let me check the commits

Reply:
fixed :)